src/libs/SidebarUtils.ts

```diff
@@ -286,7 +307,10 @@ function getOptionData(
     result.shouldShowSubscript = ReportUtils.shouldReportShowSubscript(report);
     result.pendingAction = report.pendingFields ? report.pendingFields.addWorkspaceRoom || report.pendingFields.createChat : undefined;
     result.allReportErrors = OptionsListUtils.getAllReportErrors(report, reportActions) as OnyxCommon.Errors;
-    result.brickRoadIndicator = Object.keys(result.allReportErrors ?? {}).length !== 0 ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : '';
+    result.brickRoadIndicator =
+        Object.keys(result.allReportErrors ?? {}).length !== 0 || ReportUtils.transactionThreadHasViolations(report, canUseViolations, transactionViolations, null, parentReportAction)
+            ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR
+            : '';
```

Comment:
NAB, assuming we can use `_.size`
```suggestion
    const hasViolations = ReportUtils.transactionThreadHasViolations(report, canUseViolations, transactionViolations, null, parentReportAction);
    result.brickRoadIndicator = _.size(result.allReportErrors ?? {}) !== 0 || hasViolations
            ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR
            : '';
```

Reply:
There's a linter rule against using the size underscore method.